index.html

```diff
@@ -0,0 +1,47 @@
+        <h3>Underground activity and the Rivonia Trial</h3>
+        <p>After the massacre of unarmed Black South Africans by police forces at Sharpeville in 1960 and the subsequent banning of the ANC, Mandela abandoned his nonviolent stance and began advocating acts of sabotage against the South African regime. He went underground (during which time he became known as the Black Pimpernel for his ability to evade capture) and was one of the founders of Umkhonto we Sizwe (“Spear of the Nation”), the military wing of the ANC. In 1962 he went to Algeria for training in guerrilla warfare and sabotage, returning to South Africa later that year. On August 5, shortly after his return, Mandela was arrested at a road block in Natal; he was subsequently sentenced to five years in prison.</p>
+        <h3>Incarceration</h3>
+        <p>From 1964 to 1982 Mandela was incarcerated at Robben Island Prison, off Cape Town. He was subsequently kept at the maximum-security Pollsmoor Prison until 1988, when, after being treated for tuberculosis, he was transferred to Victor Verster Prison near Paarl. The South African government periodically made conditional offers of freedom to Mandela, most notably in 1976, on the condition that he recognize the newly independent—and highly controversial—status of the Transkei Bantustan and agree to reside there. An offer made in 1985 required that he renounce the use of violence. Mandela refused both offers, the second on the premise that only free men were able to engage in such negotiations and, as a prisoner, he was not a free man.</p>
+        <h3>Presidency and retirement</h3>
+        <p>In April 1994 the Mandela-led ANC won South Africa’s first elections by universal suffrage, and on May 10 Mandela was sworn in as president of the country’s first multiethnic government. He established in 1995 the Truth and Reconciliation Commission (TRC), which investigated human rights violations under apartheid, and he introduced housing, education, and economic development initiatives designed to improve the living standards of the country’s Black population. In 1996 he oversaw the enactment of a new democratic constitution. Mandela resigned his post with the ANC in December 1997, transferring leadership of the party to his designated successor, Thabo Mbeki. Mandela and Madikizela-Mandela had divorced in 1996, and in 1998 Mandela married Graca Machel, the widow of Samora Machel, the former Mozambican president and leader of Frelimo.</p>
+      </div>
```

Comment:
This should also be put in a section tag.

Reply:
- [ ] Kindy make sure your comments are not sounding like orders but suggestions. It would be better to rephrase this comment here :smile: